sass/pagination.scss

```diff
@@ -1,13 +1,13 @@
 @import 'variables';
 
 .pagination {
-  margin-top: 50px;
+  margin-top: 25px;
 
   &__title {
     display: flex;
     text-align: center;
     position: relative;
-    margin: 100px 0 20px;
+    margin: 25px 0 20px;
 
```

Comment:
I adjusted the spacing between the comments and pagination sections so it looked nicer.